kafka-ui-react-app/src/components/ConsumerGroups/Details/ListItem.styled.ts

```diff
@@ -1,6 +1,30 @@
-import styled from 'styled-components';
+import styled, { css } from 'styled-components';
 
 export const ToggleButton = styled.td`
   padding: 8px 8px 8px 16px !important;
   width: 30px;
 `;
+
+export const TableHeaderConsumerCell = styled.th(
+  ({ title, theme: { table } }) => css`
+    font-family: Inter, sans-serif;
+    font-size: 12px;
+    font-style: normal;
+    font-weight: 400;
+    line-height: 16px;
+    letter-spacing: 0em;
+    text-align: left;
+    justify-content: start;
+    align-items: center;
+    background: ${table.th.backgroundColor.normal};
+    cursor: default;
+    color: ${table.th.color.normal};
+    padding: 4px 0 4px
+      ${(!title && 0) ||
```

Comment:
it is not a good practice to check the text value of a prop , if we want to do something with the size , we should for example pass a prop for `size` and it can several values lets say `large` `small` `medium` with normally typed in the prop `type: large | small |  medium` and choose the value of the text accordingly.

we should always make `typescript` help us rather than the opposite.

Reply:
ok